obal/data/roles/git_branch_and_commit/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: 'Get updated version from spec'
+  command: "rpmspec --query --queryformat=%{version} --srpm {{ spec_file_path }}"
```

Comment:
I wonder if this should be a module, but out of scope here.

Reply:
I think that would be good as a separate issue.